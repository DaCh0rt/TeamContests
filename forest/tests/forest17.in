945 474
3435 6693
-2760 5909
1842 -9657
5294 -7844
-2104 -3831
-5349 6085
-2141 3556
-6229 8327
6517 -5554
-6740 -304
-8434 -9355
-6099 -985
-5166 5241
-5422 -7419
4651 -3711
273 878
4194 -3485
710 3885
-8131 -8737
2966 8874
7457 -892
-6567 -615
4677 7444
-665 9329
2997 -2814
-7650 -8585
-1165 6643
-651 1989
1126 1358
9720 462
7925 -1443
862 -5867
-1477 -1374
1674 -7515
9127 9968
-6621 -1194
-8599 -6271
7077 -6288
6036 -9618
9036 140
-6565 -8165
9338 -1568
-2252 9421
9760 162
6616 -3219
234 -8904
-9018 9354
6941 2890
-2655 -108
-9810 6102
-6075 3505
9246 -7581
-4833 -2086
7267 5500
-334 5317
4260 7397
-7237 4356
6650 2656
-9905 -9327
9879 -641
8149 -8050
-6543 -8610
-6012 -9278
1729 -4671
6253 -4790
-6987 -1730
2460 -1975
3352 1787
-4212 -9647
1965 -7010
2758 -9253
8969 5380
3134 -3298
9146 360
4250 5894
-8100 4463
-9385 -9836
-6079 -1985
1615 -5858
-6857 1178
9744 -9719
-9797 -1636
-2522 -2216
7211 8250
-4135 920
3627 -9208
1593 813
4788 5969
-6003 6127
-4764 2155
-4319 7522
8869 8825
2085 4781
8794 -7296
9813 4767
-6744 -9824
-5410 946
6615 -9630
3338 -7157
4896 -3326
3930 -2423
3115 -3836
8760 2849
925 2252
9859 458
649 -5393
-4202 -6250
3062 7777
4408 -9986
-3218 1741
-8631 5163
8865 5335
-5913 -716
-8559 -2371
-6892 3309
3142 -2597
-4131 -1835
-5165 -287
1128 -9919
-9405 7583
-7320 1261
8140 1162
-4515 2184
-3063 -9307
-7521 347
-3446 -9523
-7406 7721
6242 8076
6440 3795
2138 1065
7277 -853
-1464 -592
5689 4594
-5597 4650
4308 1757
1213 7872
1044 -5275
-5974 -4220
7090 -1887
-9425 6609
-7172 9060
-663 -4639
7858 -4362
-5230 7430
6466 -2865
525 8279
289 2232
7271 -934
6155 -8313
5998 5187
2369 -9915
8422 -6221
4450 7459
489 -4319
-8256 -3549
337 6930
-2038 4806
-8814 -476
3789 -5077
10 6025
-8254 1227
2087 1550
6672 3860
1637 1521
-6620 9675
1509 -2328
5884 -1890
269 1411
-5485 1137
-2225 -3221
-6921 -2457
6210 2751
4972 -5090
2856 -3289
2653 455
1602 7784
6813 -926
-7739 685
179 -9709
3437 -6519
7225 9324
-9327 9220
9507 3340
-2585 537
-2819 9795
-7546 9872
-9308 -6583
-7672 3608
-9094 862
-7176 -753
-1442 -7432
-8726 9988
-8764 2819
-6680 4087
-9548 495
3782 8987
-6278 -8529
5419 -5050
6813 9396
-6286 5024
6692 -1254
-4544 -8115
6245 -3364
-5356 -2841
-9183 340
-7624 3042
246 3582
-1809 -3904
2152 3964
-5336 -8167
8825 3884
-1113 6562
3120 -3896
8705 5858
8913 3994
9559 9207
8974 -3310
7844 -3056
-8056 5208
6415 1891
9312 -3058
1956 5563
-9486 5321
7112 -1561
1771 3638
604 8854
6944 -1571
9177 -7762
3611 8005
-6233 242
-5637 -4057
6770 3484
-7856 -3123
4090 6293
5702 -2329
7533 -6288
8053 4489
9524 9451
-4818 -3339
-6540 1321
7634 -1300
-9151 -4233
-3660 -8618
-1973 3077
-1963 6553
-7709 3750
2049 -6606
-9191 2111
-5962 860
9809 8871
4600 319
6246 -561
-680 -2639
5151 -9840
2442 -2773
9049 7465
-185 6499
8546 -3453
8334 -3762
4934 4759
5605 311
-951 724
2340 9298
3482 -6949
-7538 9996
-2947 -5298
7489 -9573
6434 6450
4727 4161
-311 9992
4953 1193
4461 4381
-411 -649
6449 -9509
1498 2235
-1340 -629
-4694 -2603
-1112 -2963
-9394 6987
-9672 1901
1672 -1784
-6028 9110
-9508 1489
506 -9540
7909 -7684
5361 -7316
9179 -4909
5728 8720
-4368 -7134
-7167 2704
6735 3242
-6282 7615
-2616 4244
3548 -4137
459 -3313
6525 -3161
5959 5699
-9751 -7386
-9010 8571
7333 4019
-4737 2934
-5781 -597
3370 3617
6066 2225
4942 -4612
108 -3483
6513 -9482
-2272 -6197
-1883 1149
6009 -7434
-7690 6213
-4359 -5988
-9639 -1460
-6554 -7807
-9838 -1574
-487 -5188
5726 -6253
-9703 7285
2586 7317
357 -3796
-9964 5450
4585 -5265
-2956 6524
-7210 -6269
8610 4554
-4248 -6482
258 -5757
5593 3928
-2539 -7598
-426 -9957
6859 9361
-122 5966
-8053 3755
736 2031
1725 433
9725 -9886
-9391 1780
7844 -7738
-8558 -5320
5383 -3789
-8304 3577
-2301 4614
241 6252
-2722 -1877
-1702 -6887
720 4585
6859 -655
7906 5975
-7790 3516
4503 3046
-3420 -883
-9482 4462
-9790 -7143
-8787 -1824
-3914 6299
-597 9551
6515 7764
8332 9697
9556 -3562
7221 5485
1054 -7413
9631 -2101
7838 -9831
2994 -9729
2564 -6579
-7447 -9273
5644 -4625
-2260 -3888
5107 7768
9517 -3104
-3172 -1692
-4795 -9712
-3019 678
-5329 6184
-171 7574
4586 5712
5359 8301
-8530 6599
9112 -5460
3999 -3658
-6972 7627
-173 -9472
-9576 -5679
-8395 6855
-1955 7012
-5328 2467
3869 -5993
-3260 -6196
1515 -1568
8955 921
6885 4519
1178 -4988
-2548 -8562
-2401 1998
-5518 5490
-5353 1877
7790 1953
-9505 6937
-1880 -5876
4022 -1506
-4000 -5202
-4664 -4548
8952 -155
-5765 6858
-3986 3725
6781 8798
-6298 -3893
-3713 1154
-2313 -7955
2354 1500
-3477 -4219
-9640 416
6040 1008
-2999 7541
6384 8954
9522 7082
-9391 -2473
9342 -4846
9831 -7634
7045 3079
-5059 2451
5522 -6427
1658 -2691
6842 7808
-2196 -8027
9722 7396
8081 3791
3432 -9821
-2072 -7443
-9305 1065
-1335 9065
-8253 -9029
-1969 -2132
-4884 8490
2745 5505
-6066 -8469
-1422 9374
-1653 -6113
-3784 717
3761 -7131
-8961 2931
9024 8614
-5453 1503
-9203 -8219
8181 -6309
-9102 -3523
3496 -3686
4175 6961
-5259 -8337
250 5544
8418 -7210
1245 8116
-5179 9518
6788 3172
396 5507
-6149 -2771
7190 2764
8536 -6778
3149 -8577
1169 -77
5885 -8148
7781 9176
8519 -478
2559 -6465
-2043 -5528
-4786 7964
-5219 -4522
3101 9405
51 7461
-2088 -1255
2674 4921
3178 4920
7096 9682
643 -379
5886 -4373
-9325 -1768
-9770 -1410
8391 2586
4909 -9830
9789 9402
7931 1801
3242 2980
-6582 -6923
-6766 -4339
8420 2630
2222 -1289
2012 -5897
4324 4528
-7467 5899
5800 8186
-8829 -8723
3270 4704
3962 -4668
7684 8633
702 -8151
5442 -3608
5692 2858
-4243 4914
-597 9807
8785 -1517
-7312 -6877
-611 4753
-4642 -1017
-5410 -2159
5470 4159
6606 246
8765 1716
3678 -6680
9242 -5826
6066 1843
4982 -8641
2930 -6910
-6059 1801
8196 1185
-8244 3489
2432 7502
1105 5321
5839 -4686
8457 -6760
-4366 394
-5426 4418
-8316 4808
-3456 -9659
-9593 3781
-688 -3893
5870 4349
2741 -8323
440 -425
-480 -48
3179 7421
-7375 -489
4780 2028
171 -5366
-2694 -2410
1486 -9655
6182 5963
3083 -6236
5267 -8289
-7925 5691
6945 -2613
-3800 7802
7034 2214
8490 4858
4800 933
894 -1892
5619 3744
-8829 9602
5248 -2724
5690 -2107
3250 5661
7700 -8310
-4178 5215
2867 7463
1570 7681
-7426 8235
3308 8044
8092 -2860
-7798 9795
652 -3272
-4021 7461
-679 3204
3302 -218
-8604 1280
-824 -2972
-1382 4683
7460 -6844
-7635 6062
8146 8336
5222 -1625
9012 7180
-9147 -8297
-3224 -1376
3111 -2174
2418 8846
491 9314
6210 1524
-2417 3454
5404 -2947
-361 -5063
-306 1863
-8087 -4394
-6962 -1750
7834 -1698
6700 -9508
8056 9065
7669 -2680
5083 -5858
-5905 -7150
3091 7163
-2809 -6464
9221 -2996
8723 -4988
-8185 1712
-7661 -3566
-6053 1930
-5757 3930
-5855 8709
-4163 4763
8408 -3924
4161 -3649
-313 7493
-7394 -5455
-8087 3518
6059 -2879
-2151 -747
-2575 7068
8733 373
8752 -5179
-5869 7835
9684 699
-1005 8598
9808 4092
8001 -8538
4106 -6726
462 -6741
3325 -2787
-3345 2585
-2856 -96
-594 7017
7610 -883
-2570 8850
3907 6874
1268 -9076
-5391 -6000
7163 9093
7660 -4119
-7886 5459
4329 -93
-7807 2169
-9204 2289
8476 -5936
-2783 -7532
-2586 5280
1133 -1047
8614 1761
6602 9313
1241 3990
9355 7204
4677 -6142
990 3323
-8578 6500
-5410 -456
1069 6525
2094 9086
9556 3128
-6513 -8926
-7116 -7461
-9874 -5172
12 9825
3939 7557
3707 3298
4815 -8011
2420 5596
4296 -7603
-1129 -6115
5987 -1740
426 1019
478 -7376
4172 -8080
9777 -3350
290 1497
8188 6788
6430 -2877
-9579 -1075
9126 2735
3633 5907
2441 -7434
-9749 -8439
8197 -8245
-2529 6805
9891 6702
-8421 269
-5222 3229
8472 2931
7734 7515
847 -7992
-5530 -3536
-5220 3908
-7695 -1439
5161 -2238
9405 8425
-1767 -4623
-1549 -6920
-9119 -8889
3507 2634
-63 -8490
6502 -5357
-3251 -2354
-5861 -3714
1816 -3176
-3571 1127
3499 -6206
-3668 9190
-1147 -291
1728 -231
-4551 773
-5306 5470
2553 -5035
-8001 -7219
2587 6895
-1795 -6133
-5848 -1198
4673 -1285
-3852 -6166
-6561 -3281
-544 4261
-246 -902
9385 -565
1076 -8603
-7966 1949
1367 1222
7262 842
4264 7916
913 -7353
-6684 1711
-8591 -5963
8743 1601
-2371 7758
-4838 4990
-6686 -5252
1637 1313
-9668 -3760
-3591 -2690
-2539 9782
6939 3814
-3320 3045
7322 -5173
4440 -8156
-5382 9310
4788 7293
3643 8072
8645 3800
-937 9919
-9165 2023
7407 4479
-7006 9624
1457 1225
-1106 2406
3990 -1404
5490 -4588
2208 -3602
-3459 -1136
-8016 7901
-3341 3743
-266 7549
9997 -7631
-1492 6227
2932 8626
2797 -6431
-8721 6487
3031 643
1782 -7791
-9235 8100
-3097 -9202
9605 -1967
-6943 2230
5766 4822
9670 -7538
795 -1466
5832 -3542
-199 -7854
-7483 7994
-2594 6776
3756 -5374
6240 6574
9750 -5890
5305 -9570
-6025 3165
-7354 6377
6111 -2039
186 6375
-4798 6964
-1228 8511
-9466 -2855
2913 -6450
4420 -6167
7296 -8052
2407 5135
5002 -875
5273 -4394
9837 7554
-6105 -8649
-8223 6229
7944 -8617
-2795 6193
3226 6899
-9724 9128
-8032 -6123
1993 1326
-7470 -7833
9080 -9462
-8574 2005
-4964 9746
577 -1545
-7984 -2151
-4687 -5292
-2989 8329
6394 3506
-4595 -4271
3441 6395
1372 -9501
6919 -3341
136 -4922
6644 7162
-682 -8583
3992 -619
-8652 9521
4440 -7989
8135 -3222
9031 -4017
-6989 -975
8119 -8262
588 6459
1571 6448
4640 -7025
9490 8831
5693 -2500
-1442 -2539
575 -4240
2924 5969
-2312 5928
-819 2341
-4971 3814
4902 -5807
7654 -7430
-1989 344
3273 1335
-9799 -6881
-2125 5406
-8250 -870
1940 -7522
2450 3299
-8185 -2458
-5545 9238
-1499 -4239
-939 1240
6651 9283
-2352 -4213
-2934 -3645
-8169 9355
9681 -8020
559 -1654
-2215 6850
-2783 8290
4521 -3862
4122 -2587
4117 1806
-24 603
-9525 -5157
-3770 -7888
-1341 6090
-8110 -7148
-3408 -5806
-2330 -8755
-2873 -2592
-9975 -1434
-2680 -8576
9362 3083
7542 -7958
7260 -182
-3607 -6031
3293 -6448
2706 -3018
7154 3249
-2805 -9236
-6337 8315
3115 -1157
-8910 -4894
-2733 -721
1580 1476
-3641 -5309
-9987 -1513
6587 5531
-8700 -8873
5233 8162
-9268 684
5298 -4100
-238 -4868
1486 9538
-3100 -2378
8669 -8463
4210 1929
5465 5264
5703 9072
5284 -814
5424 5506
3576 -3196
2579 -8567
813 7366
7845 6056
1454 2546
-1051 -4945
9204 5227
-8904 7412
-6735 3892
-4788 -11
-9443 2964
6025 -9373
-2210 2598
-4734 8872
9366 6693
-9565 -5093
-7005 -632
9413 -7705
4328 -8001
-8103 3637
-1601 6991
7527 5208
-2035 2457
-2037 5843
-1008 -226
-6344 -8605
2431 -3277
-2984 -8609
-4892 -5589
-9814 -4224
9100 5511
329 5521
9651 -4096
-3123 -8196
9515 7571
-9607 -7468
5099 -6087
-9952 -7538
856 -4270
-6611 -5957
1132 -7950
-7456 -5149
3482 -5583
9562 9714
1182 1453
-2061 -2621
7470 -4621
1139 6806
9491 5336
-9350 -5399
-7311 -1488
-6613 5608
6503 577
9410 -2548
-1722 9683
-2884 -244
4578 -4408
-1222 -7586
5778 -9819
