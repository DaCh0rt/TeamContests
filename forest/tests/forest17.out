80648.02 431979881.14
